packages/router/src/router_state.ts

```diff
@@ -28,7 +28,8 @@ import {Tree, TreeNode} from './utils/tree';
  * and the resolved data.
  * Use the `ActivatedRoute` properties to traverse the tree from any node.
  *
- * ### Example
+ * The following fragment shows how a component gets the root node
+ * of the current state to establish its own route tree.
```

Comment:
```suggestion
 * of the current state to establish its own route tree:
```
I think we discussed this before and we realized that the colon was now recommended in the tech writing style guide?